internal/namespaces/lb/v1/testdata/test-get-frontend-simple.golden

```diff
@@ -0,0 +1,10 @@
+🎲🎲🎲 EXIT CODE: 1 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Not found
```

Comment:
not normal

Reply:
This is probable because LB does not return a standard error yet